### PR TITLE
Updated following Test App

### PR DESCRIPTION
**Pull request by chandan-anjani**
- Contoso.WPF.Demo
- Contoso.WPF.Demo.DotNetCore
- Contoso.WPF.Puppet.DotNetCore

Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A few sentences describing the overall goals of the pull request.

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: Apps/Contoso.WPF.Demo/MainWindow.xaml.cs [hunk 1]
```diff
@@ -52,6 +53,12 @@ public MainWindow()
             textAttachments = Settings.Default.TextErrorAttachments;
             TextAttachmentTextBox.Text = textAttachments;
             FileAttachmentLabel.Content = fileAttachments;
+
+            if (!string.IsNullOrEmpty(Settings.Default.UserId))
```
Comment (guperrot) [inline Apps/Contoso.WPF.Demo/MainWindow.xaml.cs]: lets keep it in same place in all apps (after country code)

Comment (chandan-anjani) [reply Apps/Contoso.WPF.Demo/MainWindow.xaml.cs]: I tried to do that. But seems like other Apps don't have CountryCode initialized in the MainWindow().

Comment (chandan-anjani) [reply Apps/Contoso.WPF.Demo/MainWindow.xaml.cs]: Moved it down.

File: Apps/Contoso.WPF.Demo.DotNetCore/Settings.settings [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version='1.0' encoding='utf-8'?>
-<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="Contoso.WPF.Demo.Properties" GeneratedClassName="Settings">
+<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="Contoso.WPF.Demo.DotNetCore" GeneratedClassName="Settings">
```
Comment (guperrot) [inline Apps/Contoso.WPF.Demo.DotNetCore/Settings.settings]: Did you test that? Shouldn't it be Properties.DotNetCore?

Comment (chandan-anjani) [reply Apps/Contoso.WPF.Demo.DotNetCore/Settings.settings]: VS did this change and i guess, this is correct change as namespace is Contoso.WPF.Demo.DotNetCore

**Pull request merged**